submarine-sdk/tests/tracking/test_utils.py

```diff
@@ -0,0 +1,62 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import mock
+import os
+from submarine.store.sqlalchemy_store import SqlAlchemyStore
+from submarine.tracking.utils import is_tracking_uri_set, _TRACKING_URI_ENV_VAR, \
+    get_tracking_uri, _JOB_NAME_ENV_VAR, get_job_name, get_sqlalchemy_store
+
+
+def test_is_tracking_uri_set():
+    env = {
+        _TRACKING_URI_ENV_VAR:
+            "mysql+pymysql://submarine:password@localhost:3306/submarineDB",
```

Comment:
use DEFAULT_SUBMARINE_SERVER_PATH instead?

Reply:
change to DEFAULT_SUBMARINE_SERVER_PATH